VideoLocker/res/layout/fragment_discussion_thread_posts.xml

```diff
@@ -1,66 +1,79 @@
-            android:layout_height="wrap_content"
-            android:layout_gravity="center_vertical"
-            android:layout_marginStart="@dimen/edx_margin"
-            android:layout_marginLeft="@dimen/edx_margin"
-            android:text="@string/discussion_posts_refine"
-            android:textColor="@color/edx_grayscale_neutral_base"/>
+        <LinearLayout
+            android:id="@+id/spinners_container"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/discussion_post_refine_layout_height"
+            android:background="@color/edx_grayscale_neutral_white"
+            android:orientation="horizontal"
+            android:visibility="gone"
+            tools:visibility="visible">
 
-        <android.support.v7.widget.AppCompatSpinner
-            style="?attr/popupSpinnerStyle"
-            android:id="@+id/discussion_posts_filter_spinner"
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:layout_marginStart="@dimen/edx_margin"
-            android:layout_marginLeft="@dimen/edx_margin"
-            android:minWidth="@dimen/discussion_posts_filter_popup_width"
-            tools:listitem="@layout/row_discussion_thread_dropdown"
-            tools:targetApi="17" />
+            <TextView
+                android:id="@+id/discussion_posts_refine_text_view"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:layout_marginLeft="@dimen/edx_margin"
+                android:layout_marginStart="@dimen/edx_margin"
+                android:text="@string/discussion_posts_refine"
+                android:textColor="@color/edx_grayscale_neutral_base" />
 
-        <android.support.v7.widget.AppCompatSpinner
-            style="?attr/popupSpinnerStyle"
-            android:id="@+id/discussion_posts_sort_spinner"
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:minWidth="@dimen/discussion_posts_filter_popup_width"
-            tools:listitem="@layout/row_discussion_thread_dropdown"
-            tools:targetApi="17" />
+            <android.support.v7.widget.AppCompatSpinner
+                android:id="@+id/discussion_posts_filter_spinner"
+                style="?attr/popupSpinnerStyle"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:layout_marginLeft="@dimen/edx_margin"
+                android:layout_marginStart="@dimen/edx_margin"
+                android:minWidth="@dimen/discussion_posts_filter_popup_width"
+                tools:listitem="@layout/row_discussion_thread_dropdown"
+                tools:targetApi="17" />
 
-    </LinearLayout>
+            <android.support.v7.widget.AppCompatSpinner
```

Comment:
By the way, the appcompat library ensures that all `Spinner` declarations in XML layouts are resolved to `AppCompatSpinner`, so there isn't really any point to defining them explicitly.